pkg/webui/components/form/index.js

```diff
@@ -13,16 +13,92 @@
 // limitations under the License.
 
 import React from 'react'
-import { Formik } from 'formik'
+import { Formik, getIn } from 'formik'
 import bind from 'autobind-decorator'
 
+import scrollIntoView from 'scroll-into-view-if-needed'
 import Notification from '../notification'
 import PropTypes from '../../lib/prop-types'
 import FormContext from './context'
 import FormField from './field'
 import FormSubmit from './submit'
 
+@bind
 class InnerForm extends React.PureComponent {
+
+  formError = React.createRef()
+  fields = {}
+  firstField = undefined
+
+  _getInvalidField () {
+    const { errors } = this.props
+
+    let field
+    for (const fieldName of Object.keys(this.fields)) {
+      const err = getIn(errors, fieldName)
+      if (err) {
+        field = this.fields[fieldName]
+        break
+      }
+    }
+
+    return field
+  }
+
+  _scroll (node, delay = 200) {
+    if (node) {
+      scrollIntoView(node, {
+        scrollMode: 'if-needed',
+        behavior: 'smooth',
+        duration: delay,
```

Comment:
```suggestion
        duration,
```
Though scroll duration doesn't really seem to work in chrome (seems to be browser dependent)
https://github.com/stipsan/scroll-into-view-if-needed#ponyfill-smooth-scrolling